### PR TITLE
Add local config to Example App to store public keys and dev environment variables

### PR DESCRIPTION
**Pull request by vault087**
There's `Config.local.example.plist` and Xcode's Build Phases `Copy Config.local.example to Config.local`.
When Xcode run and there's no `Config.local.plist` file then this script will copy default config and this file will not be added to github as it's added to .gitignore.

Having public keys and dev api urls in this public config won't require you to clean it before commit.

### CODE REVIEW

File: OmiseSDKTests/ModelTestCase.swift [hunk 1]
```diff
@@ -361,29 +361,29 @@ class ModelTestCase: XCTestCase {
...
-            shippingCountryCode: "TH",
-            shippingPostalCode: "10260",
-            name: "name surname",
-            email: "test_user@opn.ooo")), source.paymentInformation)
-            XCTAssertEqual(Flow.redirect, source.flow)
-        } catch {
-            XCTFail("Cannot decode the source \(error)")
-        }
-    }
+//    func testDecodeAtomeSource() throws {
```
Comment (ajzkk) [inline OmiseSDKTests/ModelTestCase.swift:364]: Any particular reason why you comment out this test?

**Review by ajzkk — APPROVED** [review]

File: OmiseSDK/AtomeFormViewController.swift [hunk 2]
```diff
@@ -207,11 +207,16 @@ private extension AtomeFormViewController {
             return
         }
 
+        let shippingAddress =
```
Comment (danfowler) [inline OmiseSDK/AtomeFormViewController.swift:210]: @vault087 Do you mean to have Atome changes as part of this PR?

File: OmiseSDK/PaymentChooserViewController.swift [hunk 3]
```diff
@@ -331,7 +331,10 @@ class PaymentChooserViewController: AdaptableStaticTableViewController<PaymentCh
         case ("GoToDuitNowOBWBankChooserSegue"?, let controller as DuitNowOBWBankChooserViewController):
             controller.showingValues = duitNowOBWBanks
             controller.flowSession = self.flowSession
-        default: break
+        case ("GoToAtomeSegue"?, let controller as AtomeFormViewController):
```
Comment (danfowler) [inline OmiseSDK/PaymentChooserViewController.swift:334]: @vault087 Do you mean to have Atome changes as part of this PR?

File: OmiseSDK/PaymentInformation.Atome.swift [hunk 4]
```diff
@@ -0,0 +1,60 @@
+public extension PaymentInformation {
+    /// The Atome customer information
+    struct Atome: PaymentMethod {
```
Comment (danfowler) [inline OmiseSDK/PaymentInformation.Atome.swift:3]: @vault087 Do you mean to have Atome changes as part of this PR?

File: OmiseSDKTests/PaymentInformationTestCase.swift [hunk 5]
```diff
@@ -647,12 +647,16 @@ class PaymentInformationTestCase: XCTestCase {
         let encoder = PaymentInformationTestCase.makeJSONEncoder()
 
         do {
+            let shipping = PaymentInformation.Atome.ShippingAddress(
```
Comment (danfowler) [inline OmiseSDKTests/PaymentInformationTestCase.swift:650]: @vault087 Do you mean to have Atome changes as part of this PR?

Comment (vault087) [reply OmiseSDKTests/PaymentInformationTestCase.swift:650]: Unfortunately this change accidentally come to this PR as this tasks was part of Atome. It doesn't affect Atome anyhow though so I just leave it here, it's already fixed in latest related PRs

File: OmiseSDK/AtomeFormViewController.swift (hunk 2 above)
Comment (vault087) [reply OmiseSDK/AtomeFormViewController.swift:210]: Unfortunately this change accidentally come to this PR as this tasks was part of Atome. It doesn't affect Atome anyhow though so I just leave it here, it's already fixed in latest related PRs

**Review by aashishgurung — APPROVED** [review]

**Review by danfowler — APPROVED** [review]

**Pull request merged**